### PR TITLE
Tech/upgrades

### PR DESCRIPTION
**Pull request by ndelangen**
after the cleanup

### CODE REVIEW

File: examples/angular-cli/package.json [hunk 1]
```diff
@@ -18,22 +18,22 @@
...
+    "core-js": "^2.6.5",
     "rxjs": "^6.3.3",
     "rxjs-compat": "^6.4.0",
     "zone.js": "^0.8.29"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.13.1",
-    "@angular/cli": "^7.3.2",
-    "@angular/compiler-cli": "^7.2.1",
+    "@angular-devkit/build-angular": "^0.11.4",
```
Comment (kroeder) [inline examples/angular-cli/package.json]: Why going down from `0.13.1` to `0.11.4` ?

File: examples/angular-cli/package.json [hunk 2]
```diff
@@ -57,6 +57,6 @@
     "jest-preset-angular": "^6.0.1",
     "protractor": "~5.4.2",
     "ts-node": "~8.0.2",
-    "typescript": "^3.3.1"
+    "typescript": "^3.3.3333"
```
Comment (kroeder) [inline examples/angular-cli/package.json]: this version exists and should be used? 😮

File: examples/angular-cli/package.json (hunk 1 above)
Comment (ndelangen) [reply examples/angular-cli/package.json]: typo i think

File: examples/angular-cli/package.json (hunk 2 above)
Comment (ndelangen) [reply examples/angular-cli/package.json]: There have been 3332 issues that have been patched I guess

**Review by kroeder — APPROVED** [review]

**Pull request merged**